package.json

```diff
@@ -34,12 +34,11 @@
     "eslint-config-google": "^0.14.0",
     "gts": "^2.0.2",
     "jest": "^26.4.0",
-    "mocha-param": "^2.0.1",
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "actions-on-google": "^2.12.0",
     "ajv": "^6.12.3",
-    "chai": "^4.2.0",
-    "smart-home-testing": "^1.0.0"
+    "chai": "^4.2.0"
```

Comment:
We can make this a dev dependency since it's only be used for development purposes, in the tests.